src/components/dialogs/load-creation-dialog.js

```diff
@@ -451,7 +462,12 @@ const LoadCreationDialog = ({ open, onClose, network }) => {
                             filterOptions={(options, params) => {
                                 const filtered = filter(options, params);
 
-                                if (params.inputValue !== '') {
+                                if (
+                                    params.inputValue !== '' &&
+                                    !options.find(
```

Comment:
same

Reply:
Will be done when component will be mutualized